react-frontend/src/components/Nav/navbar.js

```diff
@@ -9,17 +9,20 @@ function NavBar() {
   const routeLocation = useLocation();
   const [activePage, setActivePage] = useState(routeLocation.pathname);
   const [openMenu, setOpenMenu] = useState(false);
+  const [isMobile, setIsMobile] = useState(window.innerWidth > 800);
```

Comment:
@thegentlemanphysicist , @ryleyw  mentioned that the isMobile is a common practice using `antd`. This feels very odd to me. I'd rather see us use a framework that leverages native css or css-in-js  🤷